include/deal.II/numerics/vector_tools.templates.h

```diff
@@ -80,6 +85,131 @@ DEAL_II_NAMESPACE_OPEN
 
 namespace VectorTools
 {
+  namespace implementation
```

Comment:
We've typically called such namespaces `internal`, and changed the order of them so that `internal` is the outer namespace:
```
  namespace internal
  {
    namespace VectorTools
    {
       ...
```